essay.tex

```diff
@@ -11,33 +11,38 @@
 
-Write the main body of your essay here. Add more sections if appropriate. You may choose to write about each of your three papers in its own section, or you may choose a different structure. Either way, remember that you are being assessed on technical insight and analysis: it is not enough to merely summarise the contents of the three papers. You must demonstrate the ability to make inferences beyond what is written in the papers, and to draw the three papers together into a single coherent narrative.
+\section{Hitting the target market}
+To accurately measure the users enjoyment it is important to make sure you are using playtesters matching your target audience, otherwise findings can not be confidently applied to the game\cite{moosajee2016games}. An advised way to proceed with finding correct testers is to create user persona's that can be used when looking to hire testers, allowing for an "easier time finding representative participants for their user tests"\cite{moosajee2016games}. It is possible to combine professional testers and target market testers by using the persona method previously mentioned. The combined testers are best used for early stage testing as they will have a deeper understanding of unfinished products\cite{ollila2008using}. When towards the end of development you can look at using testers who are just in the target market and not professional to get a good idea of how the end user will respond, this leads into large group testing of the target market, where it is not always possible to have developers attend the playtests and as the test group grows it may become tricky to sort the feedback data depending on how it is set up.
```

Comment:
Use the back-tick `` ` `` to open quotation marks. This is a quirk of LaTeX that you need to accommodate e in order for quotation marks to appear correctly in the PDF.

Reply:
Please also provide spaces between the last word of a sentence and the citation. Such that `unfinished productions \cite{ollila2008using}.